app.rb

```diff
@@ -1,20 +1,135 @@
 require_relative "data"
-apartments = data[:apartments]
-tenants = data[:tenants]
+require_relative "models/apartment"
+require_relative "models/tenant"
+apartments_old = data[:apartments]
+tenants_old = data[:tenants]
+apartments = []
+tenants = []
+
+#id, address, monthly_rent, square_feet
+apartments_old.map do |unit|
+  unit_obj = Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
+  apartments << unit_obj
+end
```

Comment:
It works, but you've sort of mixed up `map` and each here. I'd say the ideal use looks like either of these:

```ruby
# Using map to create a new array
apartments = apartments_old.map do |unit|
  Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
end

# OR using each + pusing to the array

apartments_old.each do |unit|
  unit_obj = Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
  apartments << unit_obj
end
```